build.py

```diff
@@ -647,17 +660,111 @@ def BuildRegexModule( cmake, cmake_common_args, script_args ):
 
 
 def EnableCsCompleter( args ):
-  build_command = PathToFirstExistingExecutable(
-    [ 'msbuild', 'msbuild.exe', 'xbuild' ] )
-  if not build_command:
-    sys.exit( 'ERROR: msbuild or xbuild is required to build Omnisharp.' )
-
-  os.chdir( p.join( DIR_OF_THIS_SCRIPT, 'third_party', 'OmniSharpServer' ) )
-  CheckCall( [ build_command, '/property:Configuration=Release',
-                              '/property:Platform=Any CPU',
-                              '/property:TargetFrameworkVersion=v4.5' ],
-             quiet = args.quiet,
-             status_message = 'Building OmniSharp for C# completion' )
+  build_dir = p.join( DIR_OF_THIRD_PARTY, "omnisharp-roslyn" )
+  try:
+    try:
+      os.mkdir( build_dir )
+    except OSError:
+      pass
+    os.chdir( build_dir )
+
+    url_pattern = ( "https://github.com/OmniSharp/omnisharp-roslyn/"
+                    "releases/download/{0}/{1}" )
+    version = "v1.32.1"
```

Comment:
Ah I see, you're setting the version to `v1.32.1` here to work around the broken Linux/Mac builds... is it possible to override this and use a system version and `v1.32.1` still uses a pretty old version of roslyn.

Reply:
Well, if you overwrite the omnisharp-roslyn folder in third-party with your own package, it will use that instead. I assume a synlink to your system version would work as well.

Reply:
This was allegedly fixed in 1.32.12-beta.51 (which also includes FixIts!). Need to test using that version.